Tests/ClientTests/CreditCard/AccessoryViewProviderTests.swift

```diff
@@ -0,0 +1,39 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Added tests where possible. This is largely a view specific change, so the tests are minimal.